spam/spam_tutorial.py

```diff
@@ -104,11 +106,6 @@
 # Let's view a few examples.
 
 # %%
-import pandas as pd
-
-# Don't truncate text fields in the display
-pd.set_option("display.max_colwidth", 0)
```

Comment:
why did you drop this? it's not cool having a bunch of truncated examples.

Reply:
hm ok it was looking fine for me in jupyter but seeing the cut-off in the rendered html version, will add back!